### DISCUSSION TITLE
wol commands works but plugin cannot wake up Windows PC

### DISCUSSION THREAD

**Discussion by InsomniaTTT** [AlexGustafsson/homebridge-wol#219] [Questions]
Homebridge -- 1.5.0 installed in docker on my router (Ubuntu Focal Fossa (20.04.4 LTS) )
homebridge-wol -- 5.0.2

The following commands works well in my router,
wol -h 192.168.2.115 -p 9 40:B0:76:7E:B9:94

but not working triggering in the plugin

any ideas?

{
    "bridge": {
        "name": "Homebridge 4900",
        "username": "0E:A6:09:D1:95:B7",
        "port": 51212,
        "pin": "735-55-229",
        "advertiser": "avahi",
        "bind": [
            "br-lan"
        ]
    },
    "accessories": [
        {
            "name": "Razor",
            "manufacturer": "homebridge-wol",
            "model": "NetworkDevice",
            "host": "192.168.2.115",
            "pingInterval": 10,
            "pingsToChange": 2,
            "pingTimeout": 10,
            "pingCommandTimeout": 0,
            "mac": "40:B0:76:7E:B9:94",
            "broadcastAddress": "192.168.2.255",
            "startCommand": "wol -h 192.168.2.115 -p 9 40:B0:76:7E:B9:94",  //not working with or without this command.
            "startCommandTimeout": 5,
            "wakeGraceTime": 20,
            "wakeCommandTimeout": 0,
            "shutdownGraceTime": 15,
            "shutdownCommandTimeout": 0,
            "logLevel": "Debug",
            "returnEarly": true,
            "_bridge": {
                "username": "0E:D2:8D:D8:52:75",
                "port": 50141
            },
            "accessory": "NetworkDevice"
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        }
    ]
}

**Comment by AlexGustafsson** [comment]
If you turn on debug logs, do you get any more information, such as the start command failing?
If the command fails you should see a log entry like

```
HWOL-4001 An error occurred while trying to start the device. This is most likely not an issue with homebridge-wol itself. For more information see https://github.com/AlexGustafsson/homebridge-wol/wiki/Frequently-Asked-Questions#hwol-4001
```
